virtcontainers/network.go

```diff
@@ -1005,6 +1009,12 @@ func xconnectVMNetwork(endpoint Endpoint, connect bool, numCPUs uint32, disableV
 		return untapNetworkPair(endpoint)
 	case NetXConnectEnlightenedModel:
 		return fmt.Errorf("Unsupported networking model")
+	case NetXConnectNoneModel:
+		netPair.NetInterworkingModel = NetXConnectNoneModel
```

Comment:
Why setting `netPair.NetInterworkingModel` to its own value right after switching based on it?

Reply:
Just follow other cases. These codes have not been changed since they are imported. I will delete these useless code.